optimizer/plan/cost.go

```diff
@@ -87,6 +81,41 @@ func (c *costEstimator) Leave(p Plan) (Plan, bool) {
 	}
 	return p, true
 }
+
+func (c *costEstimator) tableScan(v *TableScan) {
+	var rowCount float64
+	if len(v.Ranges) == 1 && v.Ranges[0].LowVal == math.MinInt64 && v.Ranges[0].HighVal == math.MaxInt64 {
```

Comment:
Reconsidered about it, it is used only in one place, and we may change this method later, so just keep it as it is.